doc/connectivity/usb_c/ucds.rst

```diff
@@ -108,6 +108,96 @@ Start the USB-C subsystem:
    :end-before: usbc.rst usbc end
    :linenos:
 
+Implementing a Source Type-C and Power Delivery USB-C device
+************************************************************
+
+The configuration of a USB-C Device is done in the stack layer and devicetree.
+
+The following devicetree, structures and callbacks need to be defined:
+
+* Devicetree usb-c-connector node referencing a TCPC
```

Comment:
Here and below, `usb-c-connector` and `vbus` should probably have double backticks or some other markup to indicate that they are the names of DTS nodes.

Reply:
done